lib/spack/spack/installer.py

```diff
@@ -683,6 +690,18 @@ def __str__(self):
         return '{0}: {1}; {2}; {3}; {4}'.format(
             self.pid, requests, tasks, installed, failed)
 
+    def _set_term_title(self, text, force=False):
+        if not spack.config.get('config:set_terminal_title', False):
+            return
+
+        if not sys.stdout.isatty() and not force:
+            return
+
+        status = '{0} [{1}/{2}]'.format(text, self.pkg_num, self.pkg_count)
+        print('\033]0;Spack: {0}\007'.format(status), end='')
```

Comment:
Why use this instead of `sys.stdout.write`? Especially considering the number of sys.stdout calls necessary in this method.

Reply:
Good question. :smile: I probably started out with `print` and didn't think about changing it when the other `sys.stdout` calls became necessary. Fixed now.